definitions/npm/lodash_v4.x.x/flow_v0.38.x-v0.46.x/lodash_v4.x.x.js

```diff
@@ -1,3 +1,5 @@
+type Path = $ReadOnlyArray<string | number> | string | number;
```

Comment:
This defines _global_ type named `Path` that would be available in all the files without explicitly importing it. In order to make private type, you need to move this inside the module declaration.

Reply:
I see ... I thought that you have to `declare` the type for it to be global and that the bare `type` is scoped to file (given it's not exported).

The reason for which I've moved it outside of the module declaration is because it's used in `lodash/fp` as well, so this seemed cleaner this way as opposed to copying.

Are you sure about the global issues?
If so, I'll move it gladly…

Reply:
So I've changed it anyway to move along … however I'm still curious about that scoping issue

Reply:
Flow has a bit of special handling for library definitions. I went and tried out how similar definition to Path behves, and I didn't need to import it in any files. No `declaration` was needed.

Reply:
I see. Thanks for clarification!